molecule/gitlab/prepare.yml

```diff
@@ -21,6 +21,20 @@
             state: "present"
             update_cache: true
 
+        # Workaround to prevent "sudo: PAM account management error" because of non-readable shadows file on AlmaLinux
```

Comment:
How did you figure out that this workaround is needed? Is there some further reference for it?